apps/docs/package.json

```diff
@@ -11,18 +11,18 @@
     "build:animation": "node scripts/animation-support.js"
   },
   "dependencies": {
-    "@primer/gatsby-theme-doctocat": "^4.6.3",
+    "@primer/gatsby-theme-doctocat": "^5.1.1",
     "@primer/octicons-react": "^19.8.0",
-    "@primer/react": "^35.1.0",
+    "@primer/react": "36.11.0",
     "color2k": "^2.0.0",
     "cross-env": "7.0.2",
     "gatsby": "^3.7.2",
     "gatsby-plugin-alias-imports": "^1.0.5",
     "gatsby-plugin-postcss": "^4.14.0",
     "gatsby-plugin-typescript": "^2.12.1",
-    "react": "^17.0.2",
+    "react": "^18.x",
```

Comment:
Can we pin this version too please and avoid wildcards. Guessing it will be `18.2.0` here?